### DISCUSSION TITLE
OPP-010 and OPP-011

### DISCUSSION THREAD

**Discussion by PaulGambke** [OP-TED/eForms-SDK#634] [Q&A] [answered]
Hi, 
I'm a bit lost with these 2 fields.
They seem to be mandatory for all notices.
What values are we supposed to fill them with?
According to the Ted Developer Docs both fields are "Information added by OP prior to publication".
If OP adds them, why do we have to fill them?

**Comment by PaulGambke** [comment, marked as answer]
Ah nevermind, they are not mandatory.
I was filling the OPP-012 too.
If I remove all of them, the xml is valid.

**Reply by YvesJo** [reply]
Indeed. In fact, to answer the initial question, these fields are for the Publications Office only and will contain the values for the notice getting published.
